pixeltable/metadata/schema.py

```diff
@@ -154,6 +154,8 @@ class ViewMd:
 @dataclasses.dataclass
 class TableMd:
     name: str
+    tbl_id: str  # uuid.UUID
+    dir_id: str  # uuid.UUID
```

Comment:
Let's leave the dir id out of this, because that just reflects the local namespace (ie, the same table is going to reside in a different dir in the cloud namespace).

We could make the same argument for the name, but that would probably cause more downstream changes.